plugins/modules/nim.py

```diff
@@ -97,6 +97,11 @@
     - Forces action.
     type: bool
     default: no
+  boot_client:
```

Comment:
boot_client is a new argument, does it need to be added to AnsibleModule argument_spec dictionary?